sling/nlp/parser/trainer/pytorch_modules.py

```diff
@@ -367,12 +367,13 @@ def _ff_output(
       assert len(indices) == f.num
 
       for index in indices:
+        #print f.name, index
```

Comment:
You can probably remove this debug comment.

Reply:
Done.